Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PostProcessing/Tonemap.azsli

```diff
@@ -8,16 +8,42 @@
 
 #pragma once
 
+#include <Atom/Features/ColorManagement/TransformColor.azsli>
 
-real3 ApplyFilmicTonemap(real3 color)
+// Simple reinhard tone mapping algorithm based on below paper.
+// http://www.cmap.polytechnique.fr/~peyre/cours/x2005signal/hdr_photographic.pdf
+real3 TonemapReinhard(const real3 inputColor)
+{
+    return inputColor / (1.0 + inputColor);
+}
+
+// Originally written by Stephen Hill (@self_shadow)
+real3 TonemapAcesFitted(const real3 inputColor)
+{
+    // Apply RRT and ODT
+    float3 a = inputColor * (inputColor + 0.0245786f) - 0.000090537f;
+    float3 b = inputColor * (0.983729f * inputColor + 0.4329510f) + 0.238081f; 
+    return a / b;
```

Comment:
The style is inconsistent with the below. The operation is about the same, but coded differently, I believe it makes sense to write with a b c d e constants as the below, or maybe rewrite the TonemapAcesFilmic below.

BTW, there is no saturate() here, if the inputColor = 0, then the result is negative. Maybe it has some meaning, not sure.